src/rust/engine/process_executor/src/main.rs

```diff
@@ -170,6 +170,16 @@ struct Opt {
   #[structopt(long)]
   cas_root_ca_cert_file: Option<PathBuf>,
 
+  /// Path to file containing client certificates for the CAS server.
+  /// If not set, TLS will not be used when connecting to the CAS server.
```

Comment:
Do you mean TLS won't be used or "TLS with a client certificate" won't be used?

Reply:
TLS -> mTLS.